lib/cocoapods-core/podfile/target_definition.rb

```diff
@@ -468,6 +468,7 @@ def set_platform(name, target = nil)
       # @return [void]
       #
       def store_pod(name, *requirements)
+        parse_subspecs(name, requirements) # This parse method must be called first
```

Comment:
It's not 100% needed, but I feel it's better to write it down than make someone chase thru the tests next time around. The tests describe the desired end effect, the comment addresses the _why_.